src/plugins/plot/stackedPlot/StackedPlotItem.vue

```diff
@@ -223,33 +223,38 @@ export default {
                 const configId = this.openmct.objects.makeKeyString(this.childObject.identifier);
                 let config = configStore.get(configId);
                 if (!config) {
-                    const persistedConfig = this.domainObject.configuration.series.find((seriesConfig) => {
+                    let persistedConfig = this.domainObject.configuration.series.find((seriesConfig) => {
                         return this.openmct.objects.areIdsEqual(seriesConfig.identifier, this.childObject.identifier);
                     });
-                    if (persistedConfig) {
-                        config = new PlotConfigurationModel({
-                            id: configId,
-                            domainObject: {
-                                ...this.childObject,
-                                configuration: {
-                                    series: [
-                                        {
-                                            identifier: this.childObject.identifier,
-                                            ...persistedConfig.series
-                                        }
-                                    ],
-                                    yAxis: persistedConfig.yAxis
 
-                                }
-                            },
-                            openmct: this.openmct,
-                            palette: this.colorPalette,
-                            callback: (data) => {
-                                this.data = data;
-                            }
-                        });
-                        configStore.add(configId, config);
+                    if (!persistedConfig) {
+                        persistedConfig = {
+                            series: {}
+                        };
```

Comment:
do we also need a `yAxis` default here per line 247?

Reply:
Actually I don't think we even need these defaults. I'm still testing.